pages/index/-components/Wumpus.vue

```diff
@@ -7,23 +7,28 @@ export default {
   head () {
     return {
       script: [
-        { hid: 'rive', src: 'https://unpkg.com/rive-js/dist/rive.js', defer: false }
+        {
+          hid: 'rive',
+          src: 'https://unpkg.com/rive-js/dist/rive.js',
+          callback: () => {
+            // eslint-disable-next-line no-new, no-undef
+            new RiveAnimation({
+              src: '/rive/wumpus.riv',
+              canvas: this.$refs.riveWumpus,
+              autoplay: true
+            })
+          }
```

Comment:
Would this be better organised if the callback called a method?

Reply:
Yup, definitely.
It looks much cleaner